app/controllers/books_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class BooksController < CatalogController
-  add_breadcrumb "Books", :back_to_books_path, only: [ :show ]
+  add_breadcrumb "Books", :back_to_books_path, options: { id: "breadcrumbs_book", only: [ :show ] }
```

Comment:
@cdoyle-temple the `:only` field cannot be part of options:

```suggestion
  add_breadcrumb "Books", :back_to_books_path, { options: { id: "breadcrumbs_book" }, only: [ :show ] }
```

Reply:
I put them back into the hash and skipped rubocop before pushing.